README.md

```diff
@@ -32,17 +33,25 @@ You can configure the scope, which you pass in to the `provider` method via a `H
 
 * `scope`: A comma-separated list of permissions you want to request from the user. See [the Shopify API docs](http://docs.shopify.com/api/tutorials/oauth) for a full list of available permissions.
 
+* You **MUST** initiate the OmniAuth process by passing in a `shop` query parameter of the shop you’re requesting permissions for example. http://localhost:3000/auth/shopify?shop=example.myshopify.com. 
+
+* `setup`: You can provide a custom lambda which dynamically sets the site in the initializer 
```

Comment:
What use case is there for overriding this?  I think it might provide more harm than good to document this if it just makes the gem look harder to use.  If we do document this, then we should be clear that this is optional.